app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass

```

Comment:
Consider adding a custom message or additional functionality to the `OutdatedVaccineError` class to make it more informative. This can help in debugging and provide more context when the error is raised.